main.go

```diff
@@ -86,6 +86,11 @@ func main() {
 			Value: "mixed",
 			Usage: "Server mode, manager|agent|mixed ",
 		},
+		cli.StringFlag{
+			Name:  "work-dir",
+			Value: "./data/",
```

Comment:
how about make this visible? as hidden directory give me a hint that the data isn't important